libstack.go

```diff
@@ -0,0 +1,6 @@
+package libstack
+
+type Deployer interface {
+	Deploy(projectName, host string, filePaths []string, envFilePath string) error
+	Remove(projectName, host string, filePaths []string) error
```

Comment:
Function signatures should support `context` as discussed above.

```go
type Deployer interface {
	Deploy(ctx context.Context, projectName, host string, filePaths []string, envFilePath string) error
	Remove(ctx context.Context, projectName, host string, filePaths []string) error
}
```

Note: This would imply changing the relevant functions in the `wrapper.go` (and `wrapper_test.go`) aswell to implement the interface.